src/Microsoft.OData.Edm/Validation/EdmError.cs

```diff
@@ -27,10 +27,27 @@ public EdmError(EdmLocation errorLocation, EdmErrorCode errorCode, string errorM
             this.ErrorMessage = errorMessage;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the EdmError class.
+        /// </summary>
+        /// <param name="errorLocation">The location where the error occurred.</param>
+        /// <param name="errorCode">An integer code representing the error.</param>
+        /// <param name="errorMessage">A human readable message describing the error.</param>
+        /// <param name="errorLevel">A string describing the error severity.</param>
+        public EdmError(EdmLocation errorLocation, EdmErrorCode errorCode, string errorMessage, EdmErrorLevel errorLevel)
+                 : this(errorLocation, errorCode, errorMessage)

```

Comment:
The practice is to have the constructor with less parameters call the constructor with more parameters passing in the default value(s) for the extra parameters... Not the other way round....

Reply:
Resolved .